nlp_primitives/lsa.py

```diff
@@ -78,13 +78,13 @@ class LSA(TransformPrimitive):
     return_type = ColumnSchema(logical_type=Double, semantic_tags={"numeric"})
     default_value = 0
 
-    def __init__(self, random_seed=0, corpus=None, algorithm=None):
+    def __init__(self, random_seed=0, corpus=None, algorithm="randomized"):
         self.number_output_features = 2
         self.n = 2
         self.trainer = None
         self.random_seed = random_seed
         self.corpus = corpus
-        self.algorithm = algorithm or "randomized"
```

Comment:
curious if there is a difference doing it this way vs the way it was?

Reply:
It impacts how the feature names are generated. The way feature naming works, parameter values are only included in the feature name if the value is changed from the default value. 

With the previous approach the default value was `None` and the algorithm was set to `"randomzied"` inside `__init__` meaning that even though randomized was effectively the default value, it would always be displayed in the resulting feature name because it wasn't the actual default value in the function signature.

Doing it the new way means that `"randomized"` is the default value in the function signature, and the algorithm info will only show up in the feature name if it is changed to the non-default of `"arpack"`.